pages/tokens/index.tsx

```diff
@@ -164,11 +166,12 @@ export default function TokenList({ tokens }: { tokens: TokenInfo[] }) {
 }
 
 export const getStaticProps: GetStaticProps<TokenListProps> = async () => {
-  const tokens = getTokensInfo()
+  const tokens = await getTokensInfo()
 
   return {
     props: {
       tokens,
     },
+    revalidate: DATA_CACHE_TIME_SECONDS,
```

Comment:
this will ensure the data changes. It will still be performant, users won't experience delays